src/index.ts

```diff
@@ -0,0 +1,95 @@
+import { Octokit } from "@octokit/core";
+import { VERSION } from "./version";
+import { Endpoints } from "@octokit/types";
+
+const OCTOKIT_UNIQUE_ISSUE_ID_PREFIX = "octokit-unique-issue id=";
+
+type CreateOrUpdateUniqueIssueOptionsT =
+  Endpoints["POST /repos/{owner}/{repo}/issues"]["parameters"] & {
+    identifier: string;
+    update_previous?: boolean;
```

Comment:
The README still says `close_previous`, while the code implements `update_previous`. Intuitively I think `close_previous` is better, I have to enable something in order for the plugin to do something. Setting `update_previous: false` in order for the plugin to close the previous issue doesn't feel quite right.

Reply:
Yeah, I agree. I'll update.